dbms/src/DataStreams/CMakeLists.txt

```diff
@@ -11,7 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
```

Comment:
this file is useless?

Reply:
yes, we will add all `gtest*.cpp` file to `dbms_gtest_sources`, https://github.com/pingcap/tiflash/blob/fbed3eb9b09691015490ce1fd08254c309d0a1f8/dbms/CMakeLists.txt#L308-L319

Reply:
then i think these two CMakeLists.txt can also be removed

Reply:
sorry, misread...... the `dbms/src/Xxx/CMakeLists.txt` is necessary. We call `add_subdirectory(Xxx)` in `dbms/src/CMakeLists.txt`, the directory need a `CMakeLists.txt`, Otherwise, it reports

```
CMake Error at dbms/src/CMakeLists.txt:17 (add_subdirectory):
  The source directory

    /Users/lloyd/projects/tiflash/dbms/src/Core

  does not contain a CMakeLists.txt file.


CMake Error at dbms/src/CMakeLists.txt:18 (add_subdirectory):
  The source directory

    /Users/lloyd/projects/tiflash/dbms/src/DataStreams

  does not contain a CMakeLists.txt file.
```